PowerShellCommon.props

```diff
@@ -0,0 +1,22 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Product>PowerShell Core</Product>
+    <Company>Microsoft Corporation</Company>
+    <Copyright>(c) Microsoft Corporation. All rights reserved.</Copyright>
+
+    <FileVersion>6.0.0.0</FileVersion>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+
+    <ProductVersion>6.0.0-beta.2</ProductVersion>
+    <InformationalVersion>6.0.0-beta.2</InformationalVersion>
```

Comment:
beta.3 please since we created the release today. Otherwise, it looks fine to me.